pkg/util/lsof/lsof_linux_test.go

```diff
@@ -0,0 +1,634 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package lsof
+
+import (
+	"context"
+	"errors"
+	"io/fs"
+	"os"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/cihub/seelog"
+	"github.com/prometheus/procfs"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+func TestOpenFiles(t *testing.T) {
+	log.SetupLogger(seelog.Default, "debug")
```

Comment:
Do we need to setup the logger for this particular test?

Reply:
It was to help with debugging, it's not needed outside of that, I'll remove it 😄